src/pages/studyView/charts/StudyViewComponentLoader.tsx

```diff
@@ -1,30 +1,27 @@
 import * as React from "react";
-import {If} from 'react-if';
 import MobxPromise from "mobxpromise";
-import LoadingIndicator from "shared/components/loadingIndicator/LoadingIndicator"
 import styles from "../styles.module.scss";
-import classNames from 'classnames';
+import { ThreeBounce } from 'better-react-spinkit';
 
 export interface StudyViewComponentLoaderProps {
     promise: MobxPromise<any>
+    isInitialLoaded:boolean
 }
 
 export class StudyViewComponentLoader extends React.Component<StudyViewComponentLoaderProps> {
     public render() {
         return (
-            <div className={classNames(this.props.promise.isPending ? styles.studyViewAutoMargin : null, styles.studyViewLoadingIndicator)}>
-                <If condition={this.props.promise.isPending}>
-                    <LoadingIndicator
-                        isLoading={!!this.props.promise.isPending}
-                    />
-                </If>
-                <If condition={this.props.promise.isError}>
-                    <div>Error when loading data.</div>
-                </If>
-                <If condition={this.props.promise.isComplete}>
+            <div style={{ position: "relative", height: "100%"}}>
```

Comment:
Why don't we use LoadingIndicator? And this style is lost `styles.studyViewAutoMargin`

Reply:
I wasn't able to apply styles to that loader

Reply:
LoadingIndicator should have a style property which will apply styles directly to loader.  Please do use it.  Check definition.